### ISSUE TITLE
Server froze when trying to make Acetone Cyanohydrin

### ISSUE THREAD

**Issue by Penta0308** [petrolpark/Destroy#397]
**Describe the bug**

Multiplayer server froze (probably of vat reaction) when reaction happens.

**To Reproduce**

2x2 glass vat, containing a platinum powder, Nitrogen, Methane, Oxygen, (very low C2H6), Acetone, Hydrogen Cyanide.
I think I tried to put a tube of low-concentration HCl water solution into vat.
And server froze.

**Expected Behavior**

Not freezing.

**Screenshots**

None

**System Information**

Server:
Forge 47.2.32
Allocated RAM 8GB
```
$ java --version
openjdk 17.0.3 2022-04-19
OpenJDK Runtime Environment Temurin-17.0.3+7 (build 17.0.3+7)
OpenJDK 64-Bit Server VM Temurin-17.0.3+7 (build 17.0.3+7, mixed mode, sharing)
```
Server mods:
```
create-1.20.1-0.5.1.f.jar
destroy-1.20.1-0.8.jar
ae2wtlib-15.2.3-forge.jar
AI-Improvements-1.20-0.5.2.jar
almostunified-forge-1.20.1-0.9.3.jar
anvilrestoration-1.20.1-2.2.jar
appliedenergistics2-forge-15.2.1.jar
archers_paradox-1.20.1-5.0.0.17.jar
architectury-9.2.14-forge.jar
bettercombat-forge-1.8.5+1.20.1.jar
BuildingWands-mc1.20.1-2.6.9-release.jar
caelus-forge-3.2.0+1.20.1.jar
canary-mc1.20.1-0.3.3.jar
cccbridge-mc1.20.1-forge-1.6.3.jar
ccdbridge-1.0.0.jar
cc-tweaked-1.20.1-forge-1.111.0.jar
Chunk-Pregenerator-1.20-4.4.3.jar
cloth-config-11.1.118-forge.jar
cofh_core-1.20.1-11.0.2.56.jar
collective-1.20.1-7.57.jar
ConfiguredDefaults-v8.0.1-1.20.1-Forge.jar
configured-forge-1.20.1-2.2.3.jar
craft_some_elytra-forge-1.0.1.jar
createaddition-1.20.1-1.2.3.jar
createappliedkinetics-1.3.2-1.20.1.jar
create_central_kitchen-1.20.1-for-create-0.5.1.f-1.3.12.jar
create_connected-0.8.2-mc1.20.1-all.jar
createdeco-2.0.2-1.20.1-forge.jar
create-deco-additions-1.3.2.jar
createdieselgenerators-1.20.1-1.2h.jar
Create-DnDesire-1.20.1-0.1b.Release-Early-Dev.jar
create_enchantment_industry-1.20.1-for-create-0.5.1.f-1.2.9.d.jar
create_misc_and_things_ 1.20.1_4.0A.jar
create_power_loader-1.5.0-mc1.20.1.jar
create-structures-0.1.1-1.20.1-FORGE.jar
cupboard-1.20.1-2.7.jar
curios-forge-5.9.0+1.20.1.jar
do_a_barrel_roll-forge-3.5.6+1.20.1.jar
endercrop-1.20.1-1.7.0.jar
entityculling-forge-1.6.2-mc1.20.1.jar
FarmersDelight-1.20.1-1.2.4.jar
fastasyncworldsave-1.20.1-2.0.jar
ferritecore-6.0.1-forge.jar
firearrows-1.20.1-11-forge.jar
gpumemleakfix-1.20.1-1.8.jar
ImmediatelyFast-Forge-1.2.14+1.20.4.jar
Jade-1.20.1-forge-11.9.2.jar
JAOPCA-1.20.1-4.4.10.15.jar
jei-1.20.1-forge-15.3.0.4.jar
journeymap-1.20.1-5.10.0-forge.jar
JustEnoughResources-1.20.1-1.4.0.247.jar
KeepSomeInventory-v1.3.3.jar
kotlinforforge-4.10.0-all.jar
KryptonReforged-0.2.3.jar
kubejsadditions-forge-4.3.0.jar
kubejs-create-forge-2001.2.5-build.2.jar
kubejs-forge-2001.6.5-build.7.jar
kubejs-thermal-2001.1.10-build.2.jar
liquidburner-1.20.1-0.5.jar
lootjs-forge-1.20.1-2.11.0.jar
Mekanism-1.20.1-10.4.6.20.jar
MekanismAdditions-1.20.1-10.4.6.20.jar
MekanismGenerators-1.20.1-10.4.6.20.jar
MekanismTools-1.20.1-10.4.6.20.jar
merequester-forge-1.20.1-1.1.4.jar
midnightlib-forge-1.4.2.jar
modernfix-forge-5.18.1+mc1.20.1.jar
NBTac-FORGE-1.20.1-1.2.1.jar
NoChatReports-FORGE-1.20.1-v2.2.2.jar
noisium-forge-2.0.3+mc1.20-1.20.1.jar
no_nv_flash-1.20.1-1.6.0.4.jar
no-telemetry-1.8.0.jar
Patchouli-1.20.1-84-FORGE.jar
player-animation-lib-forge-1.0.2-rc1+1.20.jar
productivebees-1.20.1-12.5.1.jar
railcraft-reborn-1.20.1-1.1.4.jar
rhino-forge-2001.2.2-build.18.jar
rw_spawneggs_110_forge_for_1_20_1.jar
servercore-forge-1.5.1+1.20.1.jar
Simplest_Compression_forge_1.20.1-1.0.2.jar
Simplest_Copper_Gear_forge_1.20.1-1.0.2.jar
Simplest_Excavators_forge_1.20.1-1.1.3.jar
Simplest_Hammers_forge_1.20.1-1.1.2.jar
sliceanddice-forge-3.2.0.jar
Steam_Rails-1.6.4+forge-mc1.20.1.jar
storagedrawers-1.20.1-12.0.3.jar
TaxFreeLevels-1.3.10-forge-1.20.2.jar
thermal_dynamics-1.20.1-11.0.1.23.jar
thermal_expansion-1.20.1-11.0.1.29.jar
thermal_foundation-1.20.1-11.0.6.70.jar
thermal_integration-1.20.1-11.0.1.27.jar
treeharvester-1.20.1-8.7.jar
turtlechargingstation-1.4.2-1.20.jar
```

**Additional context**

This is stripped region file, containing the vat's content.
[region.zip](https://github.com/user-attachments/files/16201538/region.zip)

**Minecraft Username**

Penta0308

**Comment by semenishchev** [comment]
I think it would be helpful if you provided logs of that server runtime

**Comment by Penta0308** [comment]
> I think it would be helpful if you provided logs of that server runtime

I thought it, too. But there was no useful line, except `Can't keep up! Is the server overloaded? Running 28481ms or 569 ticks behind` message.

Deleting and filling the vat with nitrogen by NBTExplorer made my save work well. So, the content of region file should be enough to diagnose the problem.

**Comment by Duckchemist** [comment]
it happened to me whe i tried cyanide additon on C6H10O2 (made by reducing addipic acid)
the word froze and i had to kill the application

**Comment by petrolpark** [comment]
Duplicate of #294, fixed in 1.0
